### PR TITLE
Docs for java dropwizard

### PR DESCRIPTION
**Pull request by nickfun**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

## Description
I took a first pass at writing the docs for generating a server for Dropwizard 1.3. I followed the existing akka-http guides for structure. Also got a simple implementation here https://github.com/nickfun/demowizard that I used to learn about how it all fits together. I'm unable to build the microsite on my laptop but I did build the docs folder with Jekyll and confirmed all the pages link to each other correctly.

### CODE REVIEW

**Comment by nickfun** [comment]
@blast-hardcheese Thanks, I've corrected that

File: modules/microsite/docs/java/dropwizard/index.md [hunk 1]
```diff
@@ -22,6 +22,6 @@ Table of Contents
 Java Dropwizard
 ---------------
 
-Support for Dropwizard 1.3 has been available since guardrail v0.45.0.
+Support for Dropwizard 1.3 has been available since guardrail v0.45.0.o
```
Comment (blast-hardcheese) [inline modules/microsite/docs/java/dropwizard/index.md:25]: o

Comment (nickfun) [reply modules/microsite/docs/java/dropwizard/index.md:25]: 🤦 I don't even remember having this file open in the editor

**Comment by nickfun** [comment]
@blast-hardcheese Thanks, I've addressed your comment.

**Comment by kelnos** [comment]
@nickfun thank you *so* much for doing this.  I've been meaning to get around to writing documentation for all this stuff for a while, and I apologize that I never got around to doing it.  But the docs you've written are great, and I really appreciate it.  I bet new users will too.

**Pull request merged**